src/index.html

```diff
@@ -11,7 +11,93 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="Surname">Surname:</label>
+        <input type="text" name="Surname:"
+        id="Surname"
+          autocomplete="off" required
+        > <br>
+        <label for="Name">Name:</label>
+        <input class="input"  type="text"
+        name="Name:" id="Name"
+          autocomplete="off" required
+        ><br>
```

Comment:
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
      <fieldset>
        <legend>Personal information</legend>
        <label for="Surname">Surname:</label>
        <input
          type="text"
          name="Surname:"
          id="Surname"
          autocomplete="off"
          required
        >
        
        <label for="Name">Name:</label>
        <input
          class="input" 
          type="text"
          name="Name:"
          id="Name"
          autocomplete="off"
          required
        >
```
fix formatting everywhere
don`t use br, apply margins instead

Reply:
done